casepropods/family_connect_registration/plugin.py

```diff
@@ -20,6 +25,18 @@ class RegistrationPodConfig(PodConfig):
 
 
 class RegistrationPod(Pod):
+    def lookup_field_from_dictionaries(self, field, *lists):
+        """
+        Receives a 'field' and one or more lists of dictionaries
+        to search for the field. Returns the first match.
+        """
+        for results in lists:
```

Comment:
I think these variable names could be a bit more explicit. Like `for list in lists` ` for dict in list` `if field in dict` or something like that. Currently it's a bit difficult to figure out what you're dealing with here.